### PR TITLE
Add npm 

### PR DESCRIPTION
**Pull request by Its-Just-Nans**

### CODE REVIEW

File: wmn-data.json [hunk 1]
```diff
@@ -4388,6 +4388,16 @@
         "known" : ["dev", "pydude"],
         "cat" : "coding"
        },
+       {
+        "name" : "npm",
+        "uri_check" : "https://www.npmjs.com/~{account}",
+        "e_code" : 200,
+        "e_string" : "Packages",
```
Comment (WebBreacher) [inline wmn-data.json:4395]: This string is much too common and could appear on pages where user profiles should come back negative. Please look in the source code of the page and choose something more unique.

File: wmn-data.json [hunk 2]
```diff
@@ -4388,6 +4388,16 @@
         "known" : ["dev", "pydude"],
         "cat" : "coding"
        },
+       {
+        "name" : "npm",
+        "uri_check" : "https://www.npmjs.com/~{account}",
+        "e_code" : 200,
+        "e_string" : "Packages",
+        "m_string" : "not found",
```
Comment (WebBreacher) [inline wmn-data.json:4396]: This string is much too common and could appear on pages where user profiles should come back positive. Please look in the source code of the page and choose something more unique.

File: wmn-data.json [hunk 3]
```diff
@@ -4388,6 +4388,16 @@
         "known" : ["dev", "pydude"],
         "cat" : "coding"
        },
+       {
+        "name" : "npm",
+        "uri_check" : "https://www.npmjs.com/~{account}",
+        "e_code" : 200,
+        "e_string" : "/npm-avatar/",
+        "m_string" : "poster depicting npm's mascot wombat",
```
Comment (WebBreacher) [inline wmn-data.json:4396]: Please check this in the HTML source as I don't see this string. I see this string with HTML entities `poster depicting npm&#x27;s mascot wombat`

Comment (Its-Just-Nans) [reply wmn-data.json]: true

**Comment by WebBreacher** [comment]
And I missed that you didn't move this to the N section of the file. My bad. I'll fix it. But please, in the future, put the entries in the file according to the Name element's alphabetical position

**Comment by Its-Just-Nans** [comment]
yes sorry

I fixed it in https://github.com/WebBreacher/WhatsMyName/pull/881

**Pull request merged**